File: app/src/main/java/com/zl/weilu/androidut/bean/Person.java
package com.zl.weilu.androidut.bean;

/**
 * @Description:
 * @Author: weilu
 * @Time: 2017/11/4 10:45.
 */

public class Person {
    
    private String name;
    private int sex;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getSex() {
        return sex;
    }

    public void setSex(int sex) {
        this.sex = sex;
    }
    
    public int getAge(){
        return 11;
    }
    
    public String eat(String food){
        return food;
    }
}

File: app/src/main/java/com/zl/weilu/androidut/bean/Person.kt
package com.zl.weilu.androidut.bean

/**
 * @Description:
 * @Author: weilu
 * @Time: 2017/11/4 10:45.
 */

open class Person {

    open var name: String? = null
    open var sex: Int = 0

    open val age: Int
        get() = 11

    open fun eat(food: String?): String? {
        return food
    }
}
